cmd/controller/app/controller.go

```diff
@@ -95,6 +97,15 @@ func buildControllerContext(opts *options.ControllerOptions) (*controller.Contex
 		return nil, nil, fmt.Errorf("error creating kubernetes client: %s", err.Error())
 	}
 
+	nameservers := []string{}
+	if opts.DNS01Nameservers != "" {
+		nameservers = strings.Split(opts.DNS01Nameservers, ",")
```

Comment:
Don't need to split manually - pflag supports slices natively: https://godoc.org/github.com/spf13/pflag#StringSlice